tests/common/database_fixture.cpp

```diff
@@ -59,6 +59,8 @@ namespace graphene { namespace chain {
 using std::cout;
 using std::cerr;
 
+namespace buf = boost::unit_test::framework;
```

Comment:
`buf` is a commonly used abbreviation of `buffer`, which usually appears as a local variable or a function argument. I hope we don't use it in tests.